conf/messages.en

```diff
@@ -437,7 +437,7 @@ departure.declaration.errors.message.hyperlink = Create another departure declar
 cancellation.notification.errors.message.title   = Cancellation errors
 cancellation.notification.errors.message.heading = Cancellation errors
 cancellation.notification.errors.message.noerrors = There are one or more errors with the cancellation of departure declaration {0}.
-cancellation.notification.errors.message.elevenpluserrors = There are a number of errors with the cancellation of departure declaration {0}.
+cancellation.notification.errors.message.elevenpluserrors = There are one or more errors with the cancellation of this declaration.
```

Comment:
The key could be changed to just be something like `cancellation.notification.errors.message` to be more reflective of the actual content